frontends/p4/dontcareArgs.h

```diff
@@ -44,6 +44,14 @@ class DontcareArgs : public Transform, public ResolutionContext {
         toAdd.clear();
         return function;
     }
+    const IR::Node *postorder(IR::P4Action *action) override {
+        IR::IndexedVector<IR::StatOrDecl> body;
+        for (auto d : toAdd) body.push_back(d);
```

Comment:
```suggestion
        for (auto d : toAdd) {
            body.push_back(d);
        }
```
might be easier to read. This code was copied from the function segment, maybe change that piece of code, too?